docs/buildah-bud.md

```diff
@@ -560,6 +560,25 @@ content label. Shared volume labels allow all containers to read/write content.
 The `Z` option tells Buildah to label the content with a private unshared label.
 Only the current container can use a private volume.
 
+The `:O` flag tells buildah to mount the directory from the host as a
+temporary using the Overlay file system. The `RUN` command containers are
```

Comment:
should "temporary" be "temporary file system"?

Reply:
Switched to `temporary storage`